picoquic/picoquic_internal.h

```diff
@@ -376,6 +378,11 @@ typedef struct st_picoquic_cnx_t {
     unsigned int ack_needed : 1;
     unsigned int current_spin : 1; /* Current value of the spin bit */             
     unsigned int client_mode : 1; /* Is this connection the client side? */
+    unsigned int prev_spin : 1;  /* previous Spin bit */
+    unsigned int spin_vec : 2; 
+    unsigned int spin_edge : 1;
+    uint64_t lastTrigger;
+
```

Comment:
How are these variables initialized? Do you rely on default init to zero?

Reply:
I rely on a memset to zero in cnx creation.